src/base/bittorrent/sessionimpl.cpp

```diff
@@ -1480,6 +1468,9 @@ void SessionImpl::initializeNativeSession()
     pack.set_bool(lt::settings_pack::enable_set_file_valid_data, true);
 #endif
 
+    pack.set_str(lt::settings_pack::i2p_hostname, "127.0.0.1");
+    pack.set_int(lt::settings_pack::i2p_port, 7656);
+
```

Comment:
These lines seem totally irrelevant to the scope of the PR.

Reply:
> These lines seem totally irrelevant to the scope of the PR.

Damn, some unrelated changes were accidentally mixed in. Fixed it.